nextflow_schema.json

```
@@ -522,6 +522,59 @@
+                    "description": "Path to pre-downloaded database."
+                },
+                "protein_annotation_interproscan_applications": {
+                    "type": "string",
+                    "default": "PANTHER,ProSiteProfiles,ProSitePatterns,Pfam",
+                    "help_text": "A comma separated string specifying the database(s) to be used to annotate the coding regions annotated during the 'ANNOTATION' workflow of the pipeline. By default these include `PANTHER,ProSiteProfiles,ProSitePatterns,Pfam`.  \n- PANTHER (Protein ANalysis THrough Evolutionary Relationships): genes classified by their functions, using published scientific experimental evidence and evolutionary relationships. \n- PROSITE: protein domains, families, functional sites and specific patterns and profiles to identify them. \n- PFAM: protein families, represented by multiple sequence alignments and hidden Markov models (HMMs). \n\nThese databases were chosen based on the AMP workflow and therefore, with only these databases, do we guarantee the integration of the results to the AMPcombi final summary. \n\nNOTE: Currently, no integration of the results are implemented for the BGC and the ARG final summary tables. \n\nFor more information about all possible databases see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--applications`",
+                    "description": "Assigns the database /s to be used to annotate the coding regions."
+                },
+                "protein_annotation_interproscan_enableprecalc": {
+                    "type": "boolean",
+                    "help_text": "This increases the speed of functional annotation with InterProScan by pre-calculating matches found in the UniProtKB, thereby identifying unique matches in the query sequences for faster annotation. By default this is turned OFF.\n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `---diasable-precalc`",
+                    "description": "Pre-calculates residue mutual matches."
+                },
+                "protein_annotation_interproscan_enableresidueannot": {
+                    "type": "boolean",
+                    "help_text": "Activating this flag  will remove the residue annotation from the final XML and JSON output files, which we do not supply as output files from InerProScan. \n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--disable-residue-annot`",
+                    "description": "Enable residue annotation in XML and JSON files."
```

```suggestion
                "protein_annotation_interproscan_enableresidueannot": {
                    "type": "boolean",
                    "help_text": "Activating this flag will remove the residue annotation from the final XML and JSON output files, which we do not supply as output files from InterProScan.\n\nFor more information about this flag see the tool [documentation](https://interproscan-docs.readthedocs.io/en/latest/HowToRun.html).\n\n> Modifies tool parameter(s):\n> - InterProScan: `--disable-residue-annot`",
                    "description": "Enable residue annotation in XML and JSON files.",
                    "fa_icon": "fas fa-eraser"
```
Can you check if the logic is correct? Wouldn't it have to be `protein_annotation_interproscan_disableresidueannot`?

---

Its for other files, that we do not want to be included in teh output.